platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -278,6 +283,39 @@ static inline void _odp_pktio_tx_ts_set(pktio_entry_t *entry)
 	odp_atomic_store_u64(&entry->s.tx_ts, ts_val.u64);
 }
 
+static inline int _odp_pktio_tx_compl_enabled(pktio_entry_t *entry)
+{
+	return entry->s.enabled.tx_compl;
+}
+
+static inline void _odp_pktio_send_tx_compl_ev(pktio_entry_t *entry, odp_queue_t queue,
+					       const void *user_ptr, odp_packet_t pkt)
```

Comment:
Usage of this function would be simpler if it would be implemented in odp_packet_io.c. Prototype would contain only pktio entry and packet.

_odp_pktio_send_tx_compl_ev(pktio_entry_t *entry, odp_packet_t pkt)

Reply:
Yes, good idea, done in v2.